cms/envs/common.py

```diff
@@ -41,12 +41,8 @@
 from xmodule.modulestore.edit_info import EditInfoMixin
 
 ############################ FEATURE CONFIGURATION #############################
+STUDIO_NAME = "Studio"
```

Comment:
I wonder if you should call this CMS_NAME since not everyone will think of it as Studio (although so much of the code refers to Studio that it probably isn't worth it).